### PR TITLE
Makes supermatter not knock you over if you have magboots

### PR DESCRIPTION
**Pull request by 1fbff5f83b23d39d38b1dfcb4cac8d9b**
'Cause no fun when you get permastunned with no recourse
:cl: McDonald072
tweak: Magboots stop you from constantly falling over around a supercharged supermatter.
/:cl:

### CODE REVIEW

File: code/modules/power/supermatter/supermatter.dm [hunk 1]
```diff
@@ -676,7 +676,14 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 			var/atom/movable/pulled_object = P
 			if(ishuman(P))
 				var/mob/living/carbon/human/H = P
-				H.apply_effect(40, EFFECT_KNOCKDOWN, 0)
+				if(istype(H.shoes, /obj/item/clothing/shoes/magboots)) //yogs start
+					var/obj/item/clothing/shoes/magboots/MB = H.shoes
+					if(MB.magpulse)
+						to_chat(H, "<span class='warning'>Your \the [MB] keep you standing up against [src]'s pull!</span>")
```
Comment (Amelia0010) [inline code/modules/power/supermatter/supermatter.dm]: it should never ever ever have the, so adding the \the is dumb

Comment (ThatLing) [reply code/modules/power/supermatter/supermatter.dm]: "Your the magboots"?

Comment (Amelia0010) [reply code/modules/power/supermatter/supermatter.dm]: exactly

Comment (1fbff5f83b23d39d38b1dfcb4cac8d9b) [reply code/modules/power/supermatter/supermatter.dm]: The the is there whether or not I remove the \the, guess I forgot to remove this

**Pull request merged**